src/test/HelmetTest.js

```diff
@@ -1161,6 +1189,24 @@ describe("Helmet", () => {
                     .to.be.a("string")
                     .that.equals(`<script src="foo.js" async="" ${HELMET_ATTRIBUTE}="true"></script>`);
             });
+
+            it("won't render tag when primary attribute is null", () => {
+                ReactDOM.render(
+                    <Helmet
+                        script={[
+                            {
+                                src: undefined,
```

Comment:
Does this impact `innerHTML` support for `script`? Can you add a test for when `src` is undefined but `innerHTML` is defined?